datahub-web-react/src/graphql/browseV2.graphql

```diff
@@ -31,6 +31,16 @@ query getBrowseResultsV2($input: BrowseV2Input!) {
                         name
                     }
                 }
+                ... on Dashboard {
```

Comment:
We can probably use                 ...entityDisplayNameFields
here instead of all of these specific names!